ui/src/app/edge/live/thermometer/thermometer.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-container *ngIf="isInitialized">
+    <oe-flat-widget img="assets/img/thermometer.png" [title]="'General.temperature' | translate">
+
+        <ng-container *ngFor="let sensor of sensors, let i = index">
+            <oe-flat-widget-line [name]="sensor.alias" [channelAddress]="sensor.id + '/Temperature'"
```

Comment:
If you use the [channelAddress]-Input, you do not need to subscribe the channel in getChannelAddresses() in the .ts-file